docs/requirements.txt

```diff
@@ -3,6 +3,6 @@ sphinx-rtd-theme==1.3.0
 nbsphinx
 pandoc
 --extra-index-url https://download.pytorch.org/whl/cpu
-torch==1.13.1+cpu
+torch==2.2.0
 -f https://data.dgl.ai/wheels-internal/repo.html
 dgl==1.0.4
```

Comment:
We should also change dgl version.

Reply:
Does these version here only affect the graphstorm doc? If so, we can make it to satisfy the notebook requirement.